src/zui/l10n/messageIds.ts

```diff
@@ -109,6 +110,21 @@ export default makeMessages('zui', {
     search: m('Type to start searching'),
     searching: m('Searching...'),
   },
+  publishDate: {
+    invisible: m('Not visible, not scheduled'),
+    scheduledOnwards: m<{ startDate: string }>(
+      'Scheduled to be visible from {startDate} and onwards'
+    ),
+    scheduledWithEndDate: m<{ endDate: string; startDate: string }>(
+      'Scheduled to be visible between {startDate} and {endDate}'
+    ),
+    visibleOnwards: m<{ startDate: string }>(
+      'Visible from {startDate} onwards'
+    ),
+    visibleWithEndDate: m<{ endDate: string; startDate: string }>(
+      'Visible between {startDate} and {endDate}'
+    ),
+  },
```

Comment:
I believe this is obsolete and should be taken out?

Reply:
true, i forgot to delete this whole object in the messageIds!